src/index.json

```diff
@@ -323,7 +323,7 @@
                 "downloadUrl": "https://github.com/Azure/azure-batch-cli-extensions/releases/download/azure-batch-cli-extensions-3.0.4/azure_batch_cli_extensions-3.0.4-py2.py3-none-any.whl",
                 "filename": "azure_batch_cli_extensions-3.0.4-py2.py3-none-any.whl",
                 "metadata": {
-                    "azext.maxCliCoreVersion": "2.1.0",
```

Comment:
Why is the max 2.0.67?  cc/ @limingu

Reply:
Mainly this PR https://github.com/Azure/azure-cli/pull/9750 will break the extension load due to azure.batch.pool_operations being changed to azure.batch._pool_operations and other similar breaks in the python SDK. 

We made the mistake of not vendoring a copy of the SDK solely for the extension so this causes it to break. Specifically, this causes the CLI to install the extension properly but it can't actually load any of the methods due to invalid paths.

Judging by release timelines its probably better to be 2.0.68.